lib/netsparker/vulnerability.rb

```diff
@@ -128,13 +128,14 @@ def cleanup_html(source)
       result.gsub!(/<i>(.*?)<\/i>/) { "_#{$1}_" }
       result.gsub!(/<em>(.*?)<\/em>/) { "*#{$1}*" }
       result.gsub!(/<h2>(.*?)<\/h2>/) { "*#{$1}*" }
-      result.gsub!(/<strong>(.*?)<\/strong>/) { "*#{$1}*" }
+      result.gsub!(/<strong>(.*?)<\/strong>/) { "*#{$1}* " }
```

Comment:
What's the purpose of adding this space?

Reply:
The bolded words were getting pulled in like “**bold**next word” so this adds in a needed space.